app/components/content/content.js

```diff
@@ -321,8 +321,20 @@ const Content = ({ p2p, content, renderRow }) => {
               isLoading={isDeleting}
               onClick={async () => {
                 setIsDeleting(true)
-                await p2p.delete(content.rawJSON.url)
-                history.push('/')
+                const deleteFiles = true
+                const { response } = await remote.dialog.showMessageBox(
+                  remote.getCurrentWindow(),
+                  {
+                    type: 'warning',
+                    buttons: ['Delete all files', 'Cancel'],
+                    message: 'Do you really wanna?'
```

Comment:
This copy sounds unprofessional to me. What about
```suggestion
                    message: 'Would you also like to remove all of this module\'s files from disk? This action cannot be undone.'
```

Reply:
The user doesn't get the choice with this modal between deleting the files or just deleting the module - it's either all or nothing. Delete *all* files also sounds unnecessarily dramatic to me. I made this suggestion above:

> This will also delete these files from your Hypergraph folder. Are you sure you want to delete this content?
> Delete files / cancel

Reply:
😂 ohh hahaha i totally focused on the code i didn't think about the copy of course! 

Suggested copy looks good 👍